README.md

```diff
@@ -22,14 +22,12 @@ command in a terminal:
 flatpak install flathub org.openmsx.openMSX -y
 ```
 
-Once the Flatpak is installed, there will be icons for both openMSX and the Catapult launcher
+Once the Flatpak is installed, there will be an icon for the openMSX launcher
```

Comment:
I guess you meant to remove the word 'launcher' here. :)

Reply:
Thanks. Now if only I could see why the build 'failed'. When I follow the link it does not contain any logs or anything. Seems like the build system is busted.

In any case, most likely some linter issue, they love adding new linter rules all the time...